pool/web/view/page/page_layout.ml

```diff
@@ -307,24 +309,41 @@ module Tenant = struct
     query_language
     active_navigation
     =
+    let open Pool_context in
     let title_text = Pool_tenant.(Title.value tenant.title) in
     let page_title =
       title (txt (Format.asprintf "%s - %s" title_text "Pool Tool"))
     in
-    let custom_stylesheet =
-      link
-        ~rel:[ `Stylesheet ]
-        ~href:(Sihl.Web.externalize_path "/custom/assets/index.css")
-        ()
+    let stylesheets =
+      (* TODO: Use contect function after rebase *)
```

Comment:
open TODOs

Reply:
Yes, when PR #87 is ready, i can rebase and use those changes.

Reply:
Done